### ISSUE TITLE
opendoas raises a libc floating point exception

### ISSUE THREAD

**Issue by davidgarland** [gkisslinux/grepo#20]
```
~ $ doas ls
[   49.825687] traps: doas[161] trap divide error (...) error:0 in libc.so.6[...+166000]
Floating point exception
```

instruction pointer/stack pointer/base pointer for libc omitted as they change on each invocation-- I think only the offset should be relevant. (that being said, I tried looking at the objdump and at that position there is only a SIMD compare instruction, which can't even raise an exception like that as far as i know.... maybe I don't know what I'm doing. that seems plausible.)

oddly, I remember this working fine when I first installed it, but then the error eventually just.. started happening. If this is impossible to reproduce I wouldn't be surprised, though I'm also really curious why doas would be doing any sort of floating point math in the first place.

if there's anything else I can check on my end, let me know.

EDIT: also, sometimes the "traps:" line doesn't show and it just says "Floating point exception".

**Comment by git-bruh** [comment]
Did you update glibc after building doas ? That could be the cause, try rebuilding it.

**Comment by davidgarland** [comment]
![20210911_08h33m58s_grim](https://user-images.githubusercontent.com/10327897/132953221-db5fcfe4-c83a-4f7e-83ee-4ed3ad0228b7.png)
Seems to have a bison issue now.

**Comment by git-bruh** [comment]
The issue is fixed in bison 3.8.1, try to update that first.

**Comment by davidgarland** [comment]
Wonderful, fixed now. Thank you for the help. :)
